docs/reference/actions.md

```diff
@@ -65,11 +65,68 @@ input type=submit | click
 select            | change
 textarea          | input
 
+
+## KeyboardEvent Filter
+
+There may be cases where [KeyboardEvent](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent) Actions should only call the Controller method when certain keystrokes are used.
+
+You can install an event listener that responds only to the `Escape` key by adding `.esc` to the event name of the action descriptor, as in the following example.
+
+```html
+<div data-controller="modal"
+     data-action="keydown.esc->modal#close" tabindex="0">
+</div>
+```
+
+This will only work if the event being fired is a keyboard event.
+
+The correspondence between these filter and keys is shown below.
+
+Filter | Key Name
+-------- | --------
+enter    | Enter
+tab      | Tab
+esc      | Escape
+space    | " "
+up       | ArrowUp
+down     | ArrowDown
+left     | ArrowLeft
+right    | ArrowRight
+home     | Home
+end      | End
+[a-z]    | [a-z]
+[0-9]    | [0-9]
```

Comment:
A suggestion for refinement - using backticks on the code bits.

```markdown
| Filter  | Key Name   |
| ------- | ---------- |
| `enter` | Enter      |
| `tab`   | Tab        |
| `esc`   | Escape     |
| `space` | " "        |
| `up`    | ArrowUp    |
| `down`  | ArrowDown  |
| `left`  | ArrowLeft  |
| `right` | ArrowRight |
| `home`  | Home       |
| `end`   | End        |
| [a-z]   | [a-z]      |
| [0-9]   | [0-9]      |
```

OR - if the side borders is not really what we do (by the looks of things).

```markdown
Filter  | Key Name
------- | ----------
`enter` | Enter
`tab`   | Tab
`esc`   | Escape
`space` | " "
`up`    | ArrowUp
`down`  | ArrowDown
`left`  | ArrowLeft
`right` | ArrowRight
`home`  | Home
`end`   | End
[a-z]   | [a-z
[0-9]   | [0-9
```